library/std/src/sys/windows/ext/process.rs

```diff
@@ -98,6 +98,9 @@ pub trait CommandExt {
     /// [1]: https://docs.microsoft.com/en-us/windows/win32/procthread/process-creation-flags
     #[stable(feature = "windows_process_extensions", since = "1.16.0")]
     fn creation_flags(&mut self, flags: u32) -> &mut process::Command;
+
+    #[unstable(feature = "windows_process_extensions", since = "1.47.0")]
```

Comment:
I think you need to remove the `since` here. If there's a tracking issue, use `issue = "{issue-number}"`.

```suggestion
    #[unstable(feature = "windows_process_extensions")]
```

The error:

```
 error[E0541]: unknown meta item 'since'
   --> library/std/src/sys/windows/ext/process.rs:102:56
    |
102 |     #[unstable(feature = "windows_process_extensions", since = "1.47.0")]
    |                                                        ^^^^^^^^^^^^^^^^ expected one of `feature`, `reason`, `issue`, `soft`

error: aborting due to previous error
```